core/services/pipeline/keypath.go

```diff
@@ -0,0 +1,47 @@
+package pipeline
+
+import (
+	"bytes"
+
+	"github.com/pkg/errors"
+)
+
+var (
+	ErrWrongKeypath  = errors.New("wrong keypath format")
+	KeypathSeparator = []byte(".")
```

Comment:
Could this be configurable by ENV ?

Reply:
This could be too dangerous. If a NOP would change it by ENV, then it will make the existing specs invalid, right? I think we will have hard times investigating such incidents...